x/stake/app_test.go

```diff
@@ -147,11 +147,12 @@ func TestStakeMsgs(t *testing.T) {
 	mock.CheckBalance(t, mapp, addr2, sdk.Coins{genCoin.Minus(bondCoin)})
 	checkDelegation(t, mapp, keeper, addr2, addr1, true, sdk.NewRat(10))
 
+	// XXX add new transaction types
 	////////////////////
-	// Unbond
+	// Complete Unbonding
```

Comment:
Why commented - does this test fail?

Reply:
just needed to update tests - updated now